modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +62,18 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const copyAppropriateAttributes = <T extends HTMLElement> (original: SugarElement<T>, clone: SugarElement<T>): void => {
+  Obj.each(transferableAttributes, (_keyArray: string[], attributeName: string) =>
```

Comment:
These types shouldn't be here, as they are inferred, so if this changes later it just makes maintenance harder, as you need to change more places.

Additionally, `keyArray` should be used below instead of `transferableAttributes[attributeName]`.